src/components/banner/BannerUtils.tsx

```diff
@@ -0,0 +1,13 @@
+export enum BannerType {
+  NEUTRAL = 'neutral',
+  POSITIVE = 'positive',
+  ATTENTION = 'attention',
+  NEGATIVE = 'negative',
+}
+
+export const BannerIcon = {
+  [BannerType.NEUTRAL]: 'info-round',
```

Comment:
I think these could go directly in the styles definition
this way we put the right icon for the user according to the variant 
Still i'm wondering: what happens if I need to build my own variant with my own icon, I see i can set a custom class which is cool, but regarding icons? 
I think adding a statusIcon/variantIcon optional prop would be a nice feature

Also I see that neutral is 'info' actually i'd make it more specific language wise

Reply:
Yes it could be more flexible this way, but...
Fabio told me that we should only use the 4 defined icons, and not have something that is not defined in the design. 
Which means no custom icon and no content formatting. 😞 

IMO, the variants should be enough to handle all the use cases, and we should not have to define custom icons. This would allow to keep consistency across all the banner components used in Symphony.

Reply:
@antoinerollindev Rename this to `BannerInterfaces`

Reply:
"Also I see that neutral is 'info' actually i'd make it more specific language wise"

About that, I asked EIS team if it was better if we use `info`, `success`, `warning` and `error`.
They said we should stick to the names used in the figma. 
But if you think we should change, I can change.